playbooks/roles/streisand-client/templates/ssh-config.j2

```diff
@@ -0,0 +1,18 @@
+# Connection profile for a SOCKS proxy over a SSH connection to the forward user
```

Comment:
Totally agree - I meant to create an issue for a follow-up to migrate this config to the server's SSH role. Created https://github.com/jlund/streisand/issues/675 for this & assigned it to myself with a blocked label pending this PR landing in master.